packages/hardhat/contracts/VideoPixel.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts-upgradeable/token/ERC721/ERC721Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC721/extensions/ERC721BurnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/CountersUpgradeable.sol";
+
+
+contract VideoPixel is Initializable, ERC721Upgradeable, PausableUpgradeable, OwnableUpgradeable, ERC721BurnableUpgradeable {
+    using CountersUpgradeable for CountersUpgradeable.Counter;
+
+    CountersUpgradeable.Counter private _tokenIdCounter;
+
+    string public _baseTokenURI;
+    address pixelAddress;
+    mapping(uint256 => bool) public pixelClaimed;
+
+    /// @custom:oz-upgrades-unsafe-allow constructor
+    constructor() {
+        _disableInitializers();
+    }
+
+    function initialize(address _pixelAddress) initializer public {
+        __ERC721_init("In Doge We Trust", "IDWT");
+        __Pausable_init();
+        __Ownable_init();
+        __ERC721Burnable_init();
+        pixelAddress = _pixelAddress;
+        // merkleRoot = _merkleRoot;
+    }
+
+    function safeMint(uint256 _pixelId) public whenNotPaused {
+        // make sure address has not already claimed
+        require(!pixelClaimed[_pixelId], "Address already claimed");
```

Comment:
shall we also add a check here for if an address has claimed? we want only a single address to be able to claim once (no matter how many pixels they are holding)